src/debug/daccess/request.cpp

```diff
@@ -737,10 +744,12 @@ ClrDataAccess::GetHeapAllocData(unsigned int count, struct DacpGenerationAllocDa
     
         if (data && count >= 1)
         {
+            auto table = g_gcDacGlobals->generation_table;
             for (int i=0;i<NUMBERGENERATIONS;i++)
```

Comment:
>NUMBERGENERATIONS [](start = 27, length = 17)

I may have missed it, but if there isn't already a static_assert(DAC_NUMBERGENERATIONS == NUMBERGENERATIONS) we should add it, this loop appears to depend on that fact.